_content/booklets/heartfelt-sayings_sujato.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Heartfelt Sayings: An uplifting translation of the Udāna."
+translator: sujato
+external_url: "https://suttacentral.net/edition/ud/en/sujato?lang=en"
+drive_links:
```

Comment:
Usually we do the drive links in pdf then epub order. Also, use the new drive links. The old ones will be deleted in a few days.

- https://drive.google.com/file/d/1nR0OUJ7jCO1QLL5VDCFCReGQq2f--km5/view?usp=drivesdk
- https://drive.google.com/file/d/1vm2M98GK27ItLOqm02TEZ_i2nDb8KR5z/view?usp=drivesdk